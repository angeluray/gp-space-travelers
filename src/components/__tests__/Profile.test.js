import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import MyProfile from '../Profile';
import store from '../../redux/configureStore';

it('Check MyProfile renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <MyProfile />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
