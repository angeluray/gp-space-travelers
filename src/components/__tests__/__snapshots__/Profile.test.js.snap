// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check MyProfile renders correctly 1`] = `
<div
  className="d-flex justify-content-center mt-5 container"
>
  <div
    className="w-25 me-5 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Missions
    </h2>
    <p
      className="text-danger"
    >
      You haven't select any mission yet
    </p>
  </div>
  <div
    className="w-25 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Rockets
    </h2>
    <p
      className="text-danger"
    >
      You haven't select any rocket yet
    </p>
  </div>
</div>
`;

exports[`Test MyProfile renders correctly 1`] = `
<div
  className="d-flex justify-content-center mt-5 container"
>
  <div
    className="w-25 me-5 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Missions
    </h2>
    <p
      className="text-danger"
    >
      You haven't select any mission yet
    </p>
  </div>
  <div
    className="w-25 list-group"
    onKeyDown={[Function]}
  >
    <h2>
      My Rockets
    </h2>
    <p
      className="text-danger"
    >
      You haven't select any rocket yet
    </p>
  </div>
</div>
`;
